config/config-network.yaml

```diff
@@ -74,16 +74,16 @@ data:
     #
     istio.sidecar.includeOutboundIPRanges: "*"
 
-    # clusteringress.class specifies the default cluster ingress class
+    # ingress.class specifies the default ingress class
     # to use when not dictated by Route annotation.
     #
     # If not specified, will use the Istio ingress.
     #
-    # Note that changing the ClusterIngress class of an existing Route
+    # Note that changing the Ingress class of an existing Route
     # will result in undefined behavior.  Therefore it is best to only
     # update this value during the setup of Knative, to avoid getting
     # undefined behavior.
-    clusteringress.class: "istio.ingress.networking.knative.dev"
+    ingress.class: "istio.ingress.networking.knative.dev"
```

Comment:
This is a breaking change :(

cc @tcnghia

Reply:
We should've deprecated clusteringress.class in 0.9.   We can depreciate it now.   We will still honor clusteringress.class.  But if ingress.class is specified, clusteringress.class is ignored.     WDYT?

Reply:
+1, and let's separate that change into a different PR for ease of review.  Thanks

Reply:
Reverted.  Issue has been created to track it.  https://github.com/knative/serving/issues/5689